File: java/bubble_sort.java
public class bubble_sort{
    public static int[] bubble(int[] a){
        int n=a.length;
        for(int i=0;i<n-1;i++){
            for(int j=0;j<n-i-1;j++){
                if(a[j]>a[j+1]){
                    int temp = a[j];
                    a[j]=a[j+1];
                    a[j+1]=temp;
                }
            }
        }
        return a;
    }
    public static void main(String[] args){
        int[] b={2,41,5,3,7,5};
        int[] c=bubble(b);
        for(int i=0;i<c.length;i++){
            System.out.println(c[i]);
        }
    }
}

File: python/bubble_sort.py
class bubble_sort:
    def bubble(self,a):
        for i in range(0,len(a)-1):
            for j in range(0,len(a)-i-1):
                if(a[j]>a[j+1]):
                    temp = a[j]
                    a[j]=a[j+1]
                    a[j+1]=temp
        return a


c = bubble_sort()
a=[2,41,3,7,1,8,4]
print(c.bubble(a))